lib/pipeline/statefulPipelineStack.ts

```diff
@@ -122,24 +122,24 @@ export class StatefulPipelineStack extends cdk.Stack {
 
     // Some stack have dependencies to the 'shared stack' so we need to deploy it first beforehand
```

Comment:
I guess we could remove this comment now

Reply:
Similar thought. 
Let keep the comment there for future init another environment/account.
This comment is good.